### ISSUE TITLE
Django Admin - Skip link VoiceOver compatibility 

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1412]
**Description:**
**As a django admin user I want skip links to be compatible with my screen reader (VoiceOver)**. Currently, skip links do not seem to work properly for VoiceOver.  After closing the navigation area, when reading with VoiceOver to the next items after the nav toggle button, I am forced to go through the nav menu. This should be set to display:none or similar so that the nav is hidden completely when it is closed.

**Acceptance Criteria:** 
- [ ] All elements that skip links point to are focusable (Open/Close for Sidenav, Page h1 for main)
- [ ] Skip links work as expected when using them in combination with VoiceOver
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Create new javascript file to fix this problem and import it in `base.html` in django-admin-508
- [ ] Use JavaScript to manipulate skip links and focus
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
[HackMD notes](https://hackmd.io/3zR7vj7GQ7uidJmLR-eCYw?view#Follow-On-Stuff)
https://github.com/raft-tech/django-admin-508
https://pypi.org/project/django-admin-508/

![image](https://user-images.githubusercontent.com/28515874/140802664-a48fb074-2e2e-4e1f-a6a9-eb81c5cf7009.png)
Mockup of focus being set to close menu when user navigates to it with a skip link

![image](https://user-images.githubusercontent.com/28515874/140802756-a30a0a55-6438-41a6-9626-38ae6026309d.png)
Example of an h1 that should receive focus when user navigates to main

**Comment by jorgegonzalez** [comment]
@reitermb I could not seem to reproduce need and need your help clarifying this issue.

**Comment by reitermb** [comment]
> @reitermb I could not seem to reproduce this and need your help clarifying this issue.

Had to refresh my memory, this is just about making sure we have skip links pointing to elements that get read by Voiceover. The current ones get you to the right region but there's not a specific read that makes the focus change clear since you don't land on something like a heading that gets read to label your landing spot.

e.g. compare what happens if you skip to main on https://tdp-backend-a11y.app.cloud.gov/admin/admin_interface/theme/ to what happens if you skip to main on https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html (while running VoiceOver)

**Comment by raftmsohani** [comment]
The code changes are done on [django-admin-508](https://github.com/raft-tech/django-admin-508).
Test is done in two stages: 
1) Using a newly created django project and adding the django-admin-508 as local package  
2) Applying the changes to tdrs and test vs tdrs project

The django-admin-508 needs more maintenance from Raft. It is highly recommended to change the package name from its original fork:admin_interface.
